abrute

```diff
@@ -0,0 +1,74 @@
+.\" Manpage for abrute
+.\" Contact alcamech@gmail.com to correct errors or typos.
+.TH man 1 "12 October 2017" "1.0" "abrute man page"
```

Comment:
Change `man` to `abrute`, drop the "1.0", and change the last quoted part to just "Abrute"

Reply:
done